server/database/models/author.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Author = sequelize.define('Author', {
+    fullname: DataTypes.STRING
+  }, {});
```

Comment:
There is no 'id' property in your Authors model. It was specified in the DB diagram so it should be here also.

Reply:
The 

> There is no 'id' property in your Authors model. It was specified in the DB diagram so it should be here also.

The id property is auto generated so I don't see any need adding it here

Reply:
For the sake of clarity, it's good that you add it there since it was explicitly specified in the db diagram.

Reply:
okay

Reply:
done